### DISCUSSION TITLE
Admin UI language

### DISCUSSION THREAD

**Discussion by Arechiga13** [vendurehq/vendure#867] [Q&A]
Hello, there is a way to change the Admin UI language? We have latin americans users for the admin manage and we want to they feel comfortable.

Regards.

**Comment by michaelbromley** [comment]
Yes, in the top right menu you'll see a way to change it: 
![image](https://user-images.githubusercontent.com/6275952/117345968-7293ea80-aea7-11eb-88e4-d248b84880c1.png)

We have Spanish and Brazilian Portuguese. I think the Spanish is missing quite a few translation strings. Pull requests are very welcome to improve coverage. See [this guide](https://github.com/vendure-ecommerce/vendure/blob/master/packages/admin-ui/README.md#localization) if you would like to help.

**Reply by Arechiga13** [reply]
Excellent, I'll check and if I can collaborate with some translation I'll do it.

Thank you!
